cfglib.py

```diff
@@ -23,7 +23,11 @@
     cfg.StrOpt('temp', default='-',
                help='temporary directory on controller'),
     cfg.StrOpt('service_tenant', default='services',
-               help='Tenant name for services')
+               help='Tenant name for services'),
+    cfg.StrOpt('ssh_user', default='root',
+               help='user to connect via ssh'),
+    cfg.StrOpt('ssh_sudo_password', default='',
```

Comment:
I think default password should be `None`, I would assume "no password" is different from "empty password".